code/style.css

```diff
@@ -1,5 +1,173 @@
 /* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
 
+@import url("https://use.typekit.net/zkc3tby.css");
+
 body {
-  background-color: peachpuff;
+  background-color: #FFFF;
+  margin: 0;
+}
+
+header {
+  height: 50px;
+  background-color: #BCCECC;
+}
+
+header img {
+  width: 120px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+
+}
```

Comment:
To get the image to be in line with the navbar you could add some `margin-top` here, around 13px seems to do the trick. Then you can put the `margin-top` to 0 in your media query further down